components/Alert/Alert.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 import { cva, type VariantProps } from 'class-variance-authority';
 
 const alertVariants = cva(
-  'relative w-full rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
+  'relative w-full max-w-fit rounded-lg border p-4 [&>svg~*]:ml-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
```

Comment:
This may either have merge conflicts with Danielle's PR, or if yours goes in first, she may have conflicts with yours. Looks like you both updated the margin here. 

https://github.com/LetsGetTechnical/gridiron-survivor/pull/609

Reply:
Please remove the change from padding to margin as it is covered in PR #609

Reply:
Done.